.gitignore

```diff
@@ -45,6 +45,11 @@ coverage.xml
 # Pycharm
```

Comment:
It is not actually in the module, but in the root.  It was included to extend the ignored files, without doing a pull request just to update .gitignore.